en/django_urls/README.md

```diff
@@ -111,12 +111,12 @@ As you can see, we're now assigning a `view` called `post_list` to `^$` URL. Thi
 
 The last part `name='post_list'` is the name of the URL that will be used to identify the view. This can be the same as the name of the view but it can also be something completely different. We will be using the named URLs later in the project so it is important to name each URL in the app. We should also try to keep the names of URLs unique and easy to remember.
 
-Everything all right? Open http://127.0.0.1:8000/ in your browser to see the result.
+If you try to visit http:127.0.0.1:8000/ now, then you'll find some sort of 'web page not available' message. Take a look at your server console window.
```

Comment:
Maybe say something like "That means the server has stopped running." - it might help them to make that connection in other situations and look to the console for help.